Update _layout.twig base template

mmatuzovic opened the pull request:
1. Starting with IE11, document modes are deprecated and should no longer be used, except on a temporary basis.
https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/dev-guides/bg182625(v=vs.85)#document-mode-changes

2. Using `maximum-scale=1.0, user-scalable=no` in the viewport meta tag is considered a bad practice because it hinders users to zoom the page in some browsers.  Sites that use this setting are also not WCAG 2.1 AA compliant (https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html).

3. Using `viewport-fit=cover` by default without any CSS or without the CSS you provide causes that some of the page’s content is obscured by the device’s sensor housing, and the bottom navigation bar is very hard to use.
https://webkit.org/blog/7929/designing-websites-for-iphone-x/

### Description



### Related issues

brandonkelly commented:
I’m guessing not many people have environments where the env vars are predefined on the server _and_ they are using a `.env` file, so this seems a little too edge-case to be worth mentioning.

stijnh92 commented:
Okay no problem, I encountered this when my changes in `.env` were not being applied and spent way too much time figuring out it was already set. But you're probably right that it's an edge-case ;-)

mmatuzovic committed:
Update _layout.twig base template

1. Starting with IE11, document modes are deprecated and should no longer be used, except on a temporary basis.
https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/dev-guides/bg182625(v=vs.85)#document-mode-changes

2. Using `maximum-scale=1.0, user-scalable=no` in the viewport meta tag is considered a bad practice because it hinders users to zoom the page in some browsers.  Sites that use this setting are also not WCAG 2.1 AA compliant (https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html).

3. Using `viewport-fit=cover` by default without any CSS or without the CSS you provide causes that some of the page’s content is obscured by the device’s sensor housing, and the bottom navigation bar is very hard to use.
https://webkit.org/blog/7929/designing-websites-for-iphone-x/

mmatuzovic committed:
Make changes everywhere in the page

mattstein commented:
Thank you for such a nice PR, @mmatuzovic! Updated the [related tutorial repository](https://github.com/craftcms/tutorial-project) as well.

mmatuzovic commented:
Wow, that was quick. Thank you, @mattstein!
